pages/linux/virt-viewer.md

```diff
@@ -0,0 +1,25 @@
+# virt-viewer
+
+> Minimal graphical interface for a virtual machine (VM).
+> NOTE: 'domain' refers to the name, UUID or ID for the existing VMs (See: tldr virsh).
+> More information: <https://gitlab.com/virt-viewer/virt-viewer/-/blob/master/man/virt-viewer.pod>.
+
+- Launch virt-viewer with prompt to select running virtual machines:
+
+`virt-viewer`
+
+- Launch virt-viewer for a specific virtual machine by ID, UUID or name:
+
+`virt-viewer {{domain}}`
```

Comment:
```suggestion
`virt-viewer "{{domain}}"`
```

Optional fix: `{{domain}}` can be quoted if it can contain some special characters to POSIX-like compatible shell.